onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -270,6 +301,51 @@ Status UnpackTensor(const ONNX_NAMESPACE::TensorProto& tensor, const void* raw_d
   return Status::OK();
 }
 
+// This function doesn't support string tensors
+Status ReadExternalDataForTensor(const ONNX_NAMESPACE::TensorProto& tensor_proto,
```

Comment:
TensorProtoToMLValue is also loading an external tensor using mostly the same code. Can it be consolidated to keep just one copy?